### PR TITLE
Wrote script and endpoint to update financial summary information

### PR DESCRIPTION
**Pull request by julianrkung**
This PR:
1. Introduces an endpoint that updates a site's financial summary data given a month, year, and siteId
2. Introduces a script that calls this endpoint to update all financial summary sites. This script is going to be set to be called once per day via Heroku's cron scheduler
3. Adds a `SERVER_URL` environment variable to be filled out. The script will target the endpoint at `SERVER_URL`

### How to Review:
1. Delete the "East Asian Library" financial summary for April
2. Fill out the environment variable for `SERVER_URL` in your `.env` file. This should be pointed at your backend instance. Mee Panyar's backend instance is defaulted to run locally at `http://localhost:4000`
3. Run an instance of `meepanyar-node`
4. In another terminal, run the script via `npm run update-summaries`
5. Verify that the "East Asian Library" financial summary is created
6. Verify the values of the "East Asian Library" financial summary (Look at the Meter Readings, Payments, etc and make sure the numbers add up accurately)
7. Create a Meter Reading or Payment for an "East Asian Library" customer and rerun the script and make sure the April financial summary is appropriately updated

### CODE REVIEW

File: .env.example [hunk 1]
```diff
@@ -14,3 +14,4 @@ SALT_ROUNDS=2
 AZURE_STORAGE_CONNECTION_STRING=<AZURE_STORAGE_CONNECTION_STRING>
 AZURE_STORAGE_ACCOUNT_NAME=<AZURE_STORAGE_ACCOUNT_NAME>
 PRODUCTION_WEB_URL=
+SERVER_URL=
```
Comment (micahtyong) [inline .env.example:17]: Perhaps we can add a comment here with the default  `SERVER_URL` for future reference!

**Review by micahtyong — APPROVED** [review]
Everything looks good to me! I went through the review process with no problem (after figuring out what to set `SERVER_URL` as). For the last step, April's financial summary looked like: 
Screen Shot 2021-04-10 at 12 06 17 AM

Then, I added a new payment for a customer in East Asian Libray, reran the script, and this yielded the following result: 

Screen Shot 2021-04-10 at 12 06 29 AM

The numbers _seem_ to check out as well, although I'm unsure of the best way to verify this.

File: scripts/updateFinancialSummaryData.js [hunk 2]
```diff
@@ -0,0 +1,44 @@
+const moment = require('moment');
+const fetch = require('node-fetch');
+const { getAllSites } = require('../airtable/request');
+
+require('dotenv').config()
+
+
+const SERVER_URL = process.env.SERVER_URL;
+
+// This script is to be run once a day by Heroku Scheduler
```
Comment (micahtyong) [inline scripts/updateFinancialSummaryData.js:10]: I'm assuming the Heroku Scheduler is run from the Heroku portal and there isn't some cron job configuration that I should be looking for, right?

**Comment by julianrkung** [comment]
>I'm assuming the Heroku Scheduler is run from the Heroku portal and there isn't some cron job configuration that I should be looking for, right?

Yeah, I should actually probably explicitly mention in the code that the scheduling is done via the Heroku portal to clear any confusion, good point!

**Pull request merged**